tools/mcp/azure-sdk-python-mcp/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
```

Comment:
Orthogonal but we have been trying to move tools under the eng folder and not under the tools root folder. cc @scbedd